auth_otp/services/auth_service.py

```
@@ -0,0 +1,437 @@
+            otp_request=otp_request,
+            user=user,
+            access_token=tokens['access'],
+            refresh_token=tokens['refresh'],
+            device_id=device_id or str(uuid.uuid4()),
+            device_name=device_name or 'Unknown Device',
+            session_key=session_key or ''
+        )
+        
+        logger.info(
+            f"Verification record created for user: {user.username}, "
+            f"device: {device_name}"
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)